cmd/oras/login.go

```diff
@@ -20,36 +20,33 @@ import (
 	"context"
 	"errors"
 	"fmt"
-	"io"
 	"os"
 	"strings"
 
-	"github.com/moby/term"
 	"github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
 
+	"github.com/moby/term"
```

Comment:
nit: It would be better to put this line back to its original place or remove the empty line 28.